crates/subspace-networking/src/create/temporary_bans.rs

```diff
@@ -0,0 +1,101 @@
+use backoff::backoff::Backoff;
+use backoff::ExponentialBackoff;
+use libp2p::PeerId;
+use lru::LruCache;
+use std::num::NonZeroUsize;
+use std::ops::Add;
+use std::time::Instant;
+
+/// Details about temporary ban, used to track lifecycle of retries
+#[derive(Debug)]
+struct TemporaryBan {
+    backoff: ExponentialBackoff,
+    next_release: Option<Instant>,
```

Comment:
Looks like the ban is always permanent as of now(next_release is never set)? Why is it named temporary ban?

Reply:
My bad, fixed